composites/Plugin/SnippetEditor/components/ReplacementVariableEditor.js

```diff
@@ -67,7 +67,8 @@ class ReplacementVariableEditor extends React.Component {
 	constructor( props ) {
 		super( props );
 
-		const { content: rawContent, replacementVariables } = this.props;
+		const { content: rawContent, excludeReplaceVars } = this.props;
+		const replacementVariables = this.excludeReplaceVars( this.props.replacementVariables, excludeReplaceVars );
```

Comment:
You have removed `replacementVariables` being deconstructed from `this.props`, and call `this.props.replacementVariables`, yet include deconstruction of `excludeReplaceVars`. Nitpicky thing, but would be nice if the design patterns here are consistent.